src/test/java/tests/BaseTest.java

```diff
@@ -1,30 +1,38 @@
 package tests;
 
+import io.github.bonigarcia.wdm.WebDriverManager;
 import org.openqa.selenium.*;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.edge.EdgeDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
 import org.testng.annotations.*;
+import service.TestListener;
 
 import java.util.logging.Logger;
 
+@Listeners({TestListener.class})
+
```

Comment:
remove empty line between @ and class name

Reply:
done